services/api/src/resources/project/harborSetup.js

```diff
@@ -121,6 +121,34 @@ const createHarborOperations = (sqlClient /* : MariaSQL */) => ({
     } catch (err) {
       logger.error(`Error while setting up harbor environment variables for ${lagoonProjectName}, error: ${err}`)
     }
+
+    // Set webhooks for Harbor Project
+    if (lagoonHarborRoute === 'http://172.17.0.1:8084'){
+      var webhookAddress = 'http://172.17.0.1:7777'
+    } else {
+      var webhookAddress = "https://hooks.lagoon.amazeeio.cloud/"
+    }
+    try {
+      var res = await harborClient.post(`projects/${harborProjectID}/webhook/policies`, {
+        body: {
+          targets: [
+            {
+              type: "http",
```

Comment:
does this work as `http` if the URL is actually an `https` link?

Reply:
Yes. When created by the UI, this is always set as `http`, regardless if the route is using `http` or `https`.